bundles/binding/org.openhab.binding.zwave/database/philio/pst02.xml

```diff
@@ -0,0 +1,235 @@
+		<Class><id>0x85</id></Class>	<!-- COMMAND_CLASS_ASSOCIATION_V2-->
+		<Class><id>0x86</id></Class>	<!-- COMMAND_CLASS_VERSION_V2 -->
+		<Class><id>0x8F</id></Class>	<!-- COMMAND_CLASS_MULTI_CMD -->
+		<Class><id>0x98</id></Class>	<!-- COMMAND_CLASS_SECURITY -->
+		<Class><id>0xEF</id></Class>	<!-- COMMAND_CLASS_MARK -->
+	</CommandClasses>
+	<Configuration>
+		<Parameter>
+			<Index>2</Index>
+			<Type>byte</Type>
+			<Default>-1</Default>
+			<Minimum>-1</Minimum>
+			<Maximum>100</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Basic Set Level</Label>
+			<Help lang="en">-1	   ->  	0xFF(-1) turns on the light.
+1 - 100	   ->  	For dimmers 1 to 100 means the light strength.
+0          ->   Turn off the light.</Help>
+		</Parameter>
+		<Parameter>
+			<Index>3</Index>
+			<Type>byte</Type>
+			<Default>70</Default>
+			<Minimum>0</Minimum>
+			<Maximum>99</Maximum>
+			<Size>1</Size>
+			<Label lang="en">PIR Sensitivity</Label>
+			<Help lang="en">parameter to set the sensitivity for the PIR (Passiv Infrared Sensor)
+			0	   ->  	0 means disable the PIR motion;
+1 - 99	   ->  	1 means the lowest sensitivity, 99 means the highest sensitivity</Help>
+		</Parameter>
+		<Parameter>
+			<Index>4</Index>
+			<Type>byte</Type>
+			<Default>100</Default>
+			<Minimum>0</Minimum>
+			<Maximum>100</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Light Threshold</Label>
+			<Help lang="en">Setting the illumination threshold to turn on the light. When the event triggered and the environment illumination lower then the threshold, the device will turn on the light
+0	   ->  	Turn off illumination detected function and never turn on the light.
+1 - 99	   ->  	Illumination threshold; 1 means darkest, 99 means brightest.
+100	   ->   Turn off illumination detected function and always turn on the light.</Help>
+		</Parameter>
+		<Parameter>
+			<Index>5</Index>
+			<Type>byte</Type>
+			<Default>0</Default>
+			<Minimum>0</Minimum>
+			<Maximum>127</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Operation Mode</Label>
+			<Help lang="en">Bit 0: Reserved.
+Bit 1: 1 means enable test mode; 0 means normal mode. Notice: Ignored if DIP Switch is not set to Customer Mode.
+Bit 2: Disable the door/window function. 1: Disable, 0: Enable
+Bit 3: Set the temperature scale. 0: Fahrenheit, 1:Celsius
+Bit 4: Disable illumination report after event triggered. 1: Disable, 0: Enable
+Bit 5: Disable temperature report after event triggered. 1: Disable, 0: Enable
+Bit 6: Reserved.
+Bit 7: Disable the back key release into test mode. 1: Disable, 0: Enable</Help>
+		</Parameter>
+		<Parameter>
+			<Index>6</Index>
+			<Type>byte</Type>
+			<Default>4</Default>
+			<Minimum>0</Minimum>
+			<Maximum>127</Maximum>
+			<Size>1</Size>
+			<Label lang="en">Multi-Sensor Function Switch</Label>
+			<Help lang="en">Bit 0: Disable magnetic integrate illumination to turn on the lighting nodes in the association group 2. 1: Disable, 0: Enable
```

Comment:
This will render badly in the UI. It might be worth encapsulating it in CDATA tag and then adding some HTML to break the lines?